packages/location-management/src/components/LocationForm/index.tsx

```diff
@@ -337,7 +337,9 @@ const LocationForm = (props: LocationFormProps) => {
               showSearch
               placeholder={ENTER_A_LOCATION_GROUP_NAME_PLACEHOLDER}
               loadData={(setData) => {
-                return loadLocationTags(openSRPBaseURL, service, setData);
+                return loadLocationTags(openSRPBaseURL, service).then((response) =>
```

Comment:
Is this supposed to be part of the user groups task? If not can we handle it in a separate PR/issue?

Reply:
on it isn't really pare of this task sure let me create sperate pr for it